code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -603,7 +603,7 @@
 	attack_verb_simple = list("cut", "attack", "slash")
 	hitsound = 'sound/weapons/ego/sword1.ogg'
 	attribute_requirements = list(
-							TEMPERANCE_ATTRIBUTE = 60,
+							FORTITUDE_ATTRIBUTE = 60,
```

Comment:
Clown smiling at me is a rarely worked abno with a rarely used weapon, it should be fine to keep it as one of the rare temperance weapons